components/Player/PlayerMMR.jsx

```diff
@@ -5,15 +5,37 @@ import { connect } from 'react-redux';
 import { player } from '../../reducers';
 import styles from './PlayerHeader.css';
 
+//TODO make a dumb component
 export const PlayerMMR = ({ loading, error, rank, soloRank, mmrEstimate }) => {
   const getPlayerMMR = () => {
     if (error) return <Error />;
     if (loading) return <Spinner />;
     return (
-      <div className={styles.mmr}>
-        <div>Solo: {soloRank}</div>
-        <div>Party: {rank}</div>
-        <div>Estimate: {mmrEstimate.estimate}</div>
+      <div>
+        <span className={"text-info rating"}>
+          <span>
+            <abbr title={"Solo MMR"}>
+              <i className={"fa fa-fw fa-user"}></i>
+            </abbr>
+          </span>
+          <small>{soloRank}</small>
+        </span>
+        <span className={"text-warning rating"}>
+          <span>
+            <abbr title={"Party MMR"}>
+              <i className={"fa fa-fw fa-users"}></i>
+            </abbr>
+          </span>
+          <small>{rank}</small>
+        </span>
+        <span className={"rating"}>
+          <span>
+            <abbr title={"MMR estimate based on available data from peer players. This is an estimate of the population mean MMR of the recent matches played by this user."}>
+              <i className={"fa fa-fw fa-question-circle"}></i>
+            </abbr>
+          </span>
+          <small>{mmrEstimate.estimate}</small>
+        </span>
       </div>
```

Comment:
Also those class names won't work with css-modules, and in general it's a bad pattern to use globally named spaced names like that (that's the whole reason to use css-modules, to avoid namespace collisions and have modular css). Also, why are we even changing this? If we are trying to update the site front end, we don't have to model everything the way it was in the old site.